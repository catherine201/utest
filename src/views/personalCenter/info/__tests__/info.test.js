import React from 'react';
// import renderer from 'react-test-renderer';  // 这个不支持antdesign
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import AppSider from '..';

Enzyme.configure({ adapter: new Adapter() });
describe('(Component) AppSider -- snapshot', () => {
  it('capture snapshot of AppSider', () => {
    const footer = shallow(<AppSider />);
    expect(footer).toMatchSnapshot();
  });
});
