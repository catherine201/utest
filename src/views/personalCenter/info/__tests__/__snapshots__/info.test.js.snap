// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) AppSider -- snapshot capture snapshot of AppSider 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Info />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2
          className="add_h2 undefined"
        >
          个人信息
        </h2>,
        <Divider />,
        <div>
          <div>
            <Avatar
              prefixCls="ant-avatar"
              shape="square"
              size="default"
              src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
            />
            <Upload
              accept=""
              action="//jsonplaceholder.typicode.com/posts/"
              beforeUpload={[Function]}
              className="avatar-uploader"
              data={Object {}}
              disabled={false}
              listType="text"
              multiple={false}
              name="avatar"
              onChange={[Function]}
              prefixCls="ant-upload"
              showUploadList={false}
              supportServerRender={true}
              type="select"
            >
              <Button
                block={false}
                ghost={false}
                loading={false}
                prefixCls="ant-btn"
                type="primary"
              >
                修改头像
              </Button>
            </Upload>
          </div>
          <div>
            <div>
              <p>
                个人信息
              </p>
              <p>
                <span>
                  UID: 
                </span>
                19533333333
              </p>
              <p>
                <span>
                  昵称:
                </span>
                <Input
                  disabled={false}
                  onChange={[Function]}
                  prefixCls="ant-input"
                  type="text"
                  value="冰琉璃2050"
                />
              </p>
            </div>
            <Button
              block={false}
              ghost={false}
              loading={false}
              prefixCls="ant-btn"
              type="primary"
            >
              修改
            </Button>
          </div>
        </div>,
        <Divider />,
      ],
      "className": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "个人信息",
          "className": "add_h2 undefined",
        },
        "ref": null,
        "rendered": "个人信息",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <div>
              <Avatar
                prefixCls="ant-avatar"
                shape="square"
                size="default"
                src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
              />
              <Upload
                accept=""
                action="//jsonplaceholder.typicode.com/posts/"
                beforeUpload={[Function]}
                className="avatar-uploader"
                data={Object {}}
                disabled={false}
                listType="text"
                multiple={false}
                name="avatar"
                onChange={[Function]}
                prefixCls="ant-upload"
                showUploadList={false}
                supportServerRender={true}
                type="select"
              >
                <Button
                  block={false}
                  ghost={false}
                  loading={false}
                  prefixCls="ant-btn"
                  type="primary"
                >
                  修改头像
                </Button>
              </Upload>
            </div>,
            <div>
              <div>
                <p>
                  个人信息
                </p>
                <p>
                  <span>
                    UID: 
                  </span>
                  19533333333
                </p>
                <p>
                  <span>
                    昵称:
                  </span>
                  <Input
                    disabled={false}
                    onChange={[Function]}
                    prefixCls="ant-input"
                    type="text"
                    value="冰琉璃2050"
                  />
                </p>
              </div>
              <Button
                block={false}
                ghost={false}
                loading={false}
                prefixCls="ant-btn"
                type="primary"
              >
                修改
              </Button>
            </div>,
          ],
          "className": undefined,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <Avatar
                  prefixCls="ant-avatar"
                  shape="square"
                  size="default"
                  src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
                />,
                <Upload
                  accept=""
                  action="//jsonplaceholder.typicode.com/posts/"
                  beforeUpload={[Function]}
                  className="avatar-uploader"
                  data={Object {}}
                  disabled={false}
                  listType="text"
                  multiple={false}
                  name="avatar"
                  onChange={[Function]}
                  prefixCls="ant-upload"
                  showUploadList={false}
                  supportServerRender={true}
                  type="select"
                >
                  <Button
                    block={false}
                    ghost={false}
                    loading={false}
                    prefixCls="ant-btn"
                    type="primary"
                  >
                    修改头像
                  </Button>
                </Upload>,
              ],
              "className": undefined,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "className": undefined,
                  "prefixCls": "ant-avatar",
                  "shape": "square",
                  "size": "default",
                  "src": "https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "accept": "",
                  "action": "//jsonplaceholder.typicode.com/posts/",
                  "beforeUpload": [Function],
                  "children": <Button
                    block={false}
                    ghost={false}
                    loading={false}
                    prefixCls="ant-btn"
                    type="primary"
                  >
                    修改头像
                  </Button>,
                  "className": "avatar-uploader",
                  "data": Object {},
                  "disabled": false,
                  "listType": "text",
                  "multiple": false,
                  "name": "avatar",
                  "onChange": [Function],
                  "prefixCls": "ant-upload",
                  "showUploadList": false,
                  "supportServerRender": true,
                  "type": "select",
                },
                "ref": null,
                "rendered": Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "block": false,
                    "children": "修改头像",
                    "ghost": false,
                    "loading": false,
                    "prefixCls": "ant-btn",
                    "type": "primary",
                  },
                  "ref": null,
                  "rendered": "修改头像",
                  "type": [Function],
                },
                "type": [Function],
              },
            ],
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <div>
                  <p>
                    个人信息
                  </p>
                  <p>
                    <span>
                      UID: 
                    </span>
                    19533333333
                  </p>
                  <p>
                    <span>
                      昵称:
                    </span>
                    <Input
                      disabled={false}
                      onChange={[Function]}
                      prefixCls="ant-input"
                      type="text"
                      value="冰琉璃2050"
                    />
                  </p>
                </div>,
                <Button
                  block={false}
                  ghost={false}
                  loading={false}
                  prefixCls="ant-btn"
                  type="primary"
                >
                  修改
                </Button>,
              ],
              "className": undefined,
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    <p>
                      个人信息
                    </p>,
                    <p>
                      <span>
                        UID: 
                      </span>
                      19533333333
                    </p>,
                    <p>
                      <span>
                        昵称:
                      </span>
                      <Input
                        disabled={false}
                        onChange={[Function]}
                        prefixCls="ant-input"
                        type="text"
                        value="冰琉璃2050"
                      />
                    </p>,
                  ],
                },
                "ref": null,
                "rendered": Array [
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": "个人信息",
                    },
                    "ref": null,
                    "rendered": "个人信息",
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <span>
                          UID: 
                        </span>,
                        "19533333333",
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "UID: ",
                        },
                        "ref": null,
                        "rendered": "UID: ",
                        "type": "span",
                      },
                      "19533333333",
                    ],
                    "type": "p",
                  },
                  Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "host",
                    "props": Object {
                      "children": Array [
                        <span>
                          昵称:
                        </span>,
                        <Input
                          disabled={false}
                          onChange={[Function]}
                          prefixCls="ant-input"
                          type="text"
                          value="冰琉璃2050"
                        />,
                      ],
                    },
                    "ref": null,
                    "rendered": Array [
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "host",
                        "props": Object {
                          "children": "昵称:",
                        },
                        "ref": null,
                        "rendered": "昵称:",
                        "type": "span",
                      },
                      Object {
                        "instance": null,
                        "key": undefined,
                        "nodeType": "class",
                        "props": Object {
                          "className": undefined,
                          "disabled": false,
                          "onChange": [Function],
                          "prefixCls": "ant-input",
                          "type": "text",
                          "value": "冰琉璃2050",
                        },
                        "ref": null,
                        "rendered": null,
                        "type": [Function],
                      },
                    ],
                    "type": "p",
                  },
                ],
                "type": "div",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "block": false,
                  "children": "修改",
                  "ghost": false,
                  "loading": false,
                  "prefixCls": "ant-btn",
                  "type": "primary",
                },
                "ref": null,
                "rendered": "修改",
                "type": [Function],
              },
            ],
            "type": "div",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2
            className="add_h2 undefined"
          >
            个人信息
          </h2>,
          <Divider />,
          <div>
            <div>
              <Avatar
                prefixCls="ant-avatar"
                shape="square"
                size="default"
                src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
              />
              <Upload
                accept=""
                action="//jsonplaceholder.typicode.com/posts/"
                beforeUpload={[Function]}
                className="avatar-uploader"
                data={Object {}}
                disabled={false}
                listType="text"
                multiple={false}
                name="avatar"
                onChange={[Function]}
                prefixCls="ant-upload"
                showUploadList={false}
                supportServerRender={true}
                type="select"
              >
                <Button
                  block={false}
                  ghost={false}
                  loading={false}
                  prefixCls="ant-btn"
                  type="primary"
                >
                  修改头像
                </Button>
              </Upload>
            </div>
            <div>
              <div>
                <p>
                  个人信息
                </p>
                <p>
                  <span>
                    UID: 
                  </span>
                  19533333333
                </p>
                <p>
                  <span>
                    昵称:
                  </span>
                  <Input
                    disabled={false}
                    onChange={[Function]}
                    prefixCls="ant-input"
                    type="text"
                    value="冰琉璃2050"
                  />
                </p>
              </div>
              <Button
                block={false}
                ghost={false}
                loading={false}
                prefixCls="ant-btn"
                type="primary"
              >
                修改
              </Button>
            </div>
          </div>,
          <Divider />,
        ],
        "className": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "个人信息",
            "className": "add_h2 undefined",
          },
          "ref": null,
          "rendered": "个人信息",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <div>
                <Avatar
                  prefixCls="ant-avatar"
                  shape="square"
                  size="default"
                  src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
                />
                <Upload
                  accept=""
                  action="//jsonplaceholder.typicode.com/posts/"
                  beforeUpload={[Function]}
                  className="avatar-uploader"
                  data={Object {}}
                  disabled={false}
                  listType="text"
                  multiple={false}
                  name="avatar"
                  onChange={[Function]}
                  prefixCls="ant-upload"
                  showUploadList={false}
                  supportServerRender={true}
                  type="select"
                >
                  <Button
                    block={false}
                    ghost={false}
                    loading={false}
                    prefixCls="ant-btn"
                    type="primary"
                  >
                    修改头像
                  </Button>
                </Upload>
              </div>,
              <div>
                <div>
                  <p>
                    个人信息
                  </p>
                  <p>
                    <span>
                      UID: 
                    </span>
                    19533333333
                  </p>
                  <p>
                    <span>
                      昵称:
                    </span>
                    <Input
                      disabled={false}
                      onChange={[Function]}
                      prefixCls="ant-input"
                      type="text"
                      value="冰琉璃2050"
                    />
                  </p>
                </div>
                <Button
                  block={false}
                  ghost={false}
                  loading={false}
                  prefixCls="ant-btn"
                  type="primary"
                >
                  修改
                </Button>
              </div>,
            ],
            "className": undefined,
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <Avatar
                    prefixCls="ant-avatar"
                    shape="square"
                    size="default"
                    src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
                  />,
                  <Upload
                    accept=""
                    action="//jsonplaceholder.typicode.com/posts/"
                    beforeUpload={[Function]}
                    className="avatar-uploader"
                    data={Object {}}
                    disabled={false}
                    listType="text"
                    multiple={false}
                    name="avatar"
                    onChange={[Function]}
                    prefixCls="ant-upload"
                    showUploadList={false}
                    supportServerRender={true}
                    type="select"
                  >
                    <Button
                      block={false}
                      ghost={false}
                      loading={false}
                      prefixCls="ant-btn"
                      type="primary"
                    >
                      修改头像
                    </Button>
                  </Upload>,
                ],
                "className": undefined,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "className": undefined,
                    "prefixCls": "ant-avatar",
                    "shape": "square",
                    "size": "default",
                    "src": "https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png",
                  },
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "accept": "",
                    "action": "//jsonplaceholder.typicode.com/posts/",
                    "beforeUpload": [Function],
                    "children": <Button
                      block={false}
                      ghost={false}
                      loading={false}
                      prefixCls="ant-btn"
                      type="primary"
                    >
                      修改头像
                    </Button>,
                    "className": "avatar-uploader",
                    "data": Object {},
                    "disabled": false,
                    "listType": "text",
                    "multiple": false,
                    "name": "avatar",
                    "onChange": [Function],
                    "prefixCls": "ant-upload",
                    "showUploadList": false,
                    "supportServerRender": true,
                    "type": "select",
                  },
                  "ref": null,
                  "rendered": Object {
                    "instance": null,
                    "key": undefined,
                    "nodeType": "class",
                    "props": Object {
                      "block": false,
                      "children": "修改头像",
                      "ghost": false,
                      "loading": false,
                      "prefixCls": "ant-btn",
                      "type": "primary",
                    },
                    "ref": null,
                    "rendered": "修改头像",
                    "type": [Function],
                  },
                  "type": [Function],
                },
              ],
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <div>
                    <p>
                      个人信息
                    </p>
                    <p>
                      <span>
                        UID: 
                      </span>
                      19533333333
                    </p>
                    <p>
                      <span>
                        昵称:
                      </span>
                      <Input
                        disabled={false}
                        onChange={[Function]}
                        prefixCls="ant-input"
                        type="text"
                        value="冰琉璃2050"
                      />
                    </p>
                  </div>,
                  <Button
                    block={false}
                    ghost={false}
                    loading={false}
                    prefixCls="ant-btn"
                    type="primary"
                  >
                    修改
                  </Button>,
                ],
                "className": undefined,
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      <p>
                        个人信息
                      </p>,
                      <p>
                        <span>
                          UID: 
                        </span>
                        19533333333
                      </p>,
                      <p>
                        <span>
                          昵称:
                        </span>
                        <Input
                          disabled={false}
                          onChange={[Function]}
                          prefixCls="ant-input"
                          type="text"
                          value="冰琉璃2050"
                        />
                      </p>,
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": "个人信息",
                      },
                      "ref": null,
                      "rendered": "个人信息",
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          <span>
                            UID: 
                          </span>,
                          "19533333333",
                        ],
                      },
                      "ref": null,
                      "rendered": Array [
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "UID: ",
                          },
                          "ref": null,
                          "rendered": "UID: ",
                          "type": "span",
                        },
                        "19533333333",
                      ],
                      "type": "p",
                    },
                    Object {
                      "instance": null,
                      "key": undefined,
                      "nodeType": "host",
                      "props": Object {
                        "children": Array [
                          <span>
                            昵称:
                          </span>,
                          <Input
                            disabled={false}
                            onChange={[Function]}
                            prefixCls="ant-input"
                            type="text"
                            value="冰琉璃2050"
                          />,
                        ],
                      },
                      "ref": null,
                      "rendered": Array [
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "host",
                          "props": Object {
                            "children": "昵称:",
                          },
                          "ref": null,
                          "rendered": "昵称:",
                          "type": "span",
                        },
                        Object {
                          "instance": null,
                          "key": undefined,
                          "nodeType": "class",
                          "props": Object {
                            "className": undefined,
                            "disabled": false,
                            "onChange": [Function],
                            "prefixCls": "ant-input",
                            "type": "text",
                            "value": "冰琉璃2050",
                          },
                          "ref": null,
                          "rendered": null,
                          "type": [Function],
                        },
                      ],
                      "type": "p",
                    },
                  ],
                  "type": "div",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "class",
                  "props": Object {
                    "block": false,
                    "children": "修改",
                    "ghost": false,
                    "loading": false,
                    "prefixCls": "ant-btn",
                    "type": "primary",
                  },
                  "ref": null,
                  "rendered": "修改",
                  "type": [Function],
                },
              ],
              "type": "div",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
